src/lib/ngsi10/NotifyContextRequest.cpp

```diff
@@ -61,6 +64,46 @@ std::string NotifyContextRequest::render(ConnectionInfo* ciP, RequestType reques
 
 /* ****************************************************************************
 *
+* NotifyContextRequest::toJson -
+*/
+std::string NotifyContextRequest::toJson(ConnectionInfo* ciP, NotificationFormat notifyFormat)
```

Comment:
"Cascade comment" :)

removing ciP in CER toJson() -> removing ciP in CER vector toJson() -> removing ciP in NotifyContextRquest toJson()